event/views.py

```diff
@@ -0,0 +1,65 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from event.models import Event
+from group.models import Group
+import json
+from django.views.decorators.csrf import csrf_exempt
+from ast import literal_eval
+# Create your views here.
+
+@csrf_exempt
+def get_event_by_id(request, event_id):
+    if request.method == 'GET':
+        try:
+            event = Event.objects.get(event_id=event_id)
+            response = json.dumps({'event_id': event.event_id, 'event_name': event.name, 'budget': event.budget, 'activities': event.activities})
+        except Exception as e:
+            response = json.dumps({'Error': str(e)})
+        return HttpResponse(response, content_type='text/json')
+
+@csrf_exempt
+def create_event(request):
+    if request.method == 'POST':
+        payload = json.loads(request.body)
+        event_name_input = payload["event_name"]
+        group_id_input = payload["group_id"]
```

Comment:
I don't think we want to pass `group_id` as part of the payload

Reply:
we need to be able to tell the event what group they should belong to no?